src/pages/home/report/ReportActionsList.tsx

```diff
@@ -532,24 +541,27 @@ function ReportActionsList({
         // show marker based on report.lastReadTime
         const newMessageTimeReference = lastMessageTime.current && report.lastReadTime && lastMessageTime.current > report.lastReadTime ? userActiveSince.current : report.lastReadTime;
         lastMessageTime.current = null;
-        if (
-            scrollingVerticalOffset.current >= MSG_VISIBLE_THRESHOLD ||
-            !sortedVisibleReportActions.some(
-                (reportAction) =>
-                    newMessageTimeReference &&
-                    newMessageTimeReference < reportAction.created &&
-                    (ReportActionsUtils.isReportPreviewAction(reportAction) ? reportAction.childLastActorAccountID : reportAction.actorAccountID) !== Report.getCurrentUserAccountID(),
-            )
-        ) {
+        const areSomeReportActionsUnread = sortedVisibleReportActions.some((reportAction) => {
+            /**
+             * The archived reports should not be marked as unread. So we are checking if the report is archived or not.
+             * If the report is archived, we will mark the report as read.
+             */
+            const isArchivedReport = ReportUtils.isArchivedRoom(report);
+            const isUnread = isArchivedReport || (newMessageTimeReference && newMessageTimeReference < reportAction.created);
```

Comment:
@mkhutornyi No this won't cause any regression. The reason is that this code will only be executed when we click on the LHN item. So if we fix the "Mark as unread" bug in future, it would mean that we will be able to right click on the LHN item and mark that as unread, which is irrespective of the changes here.

Also, since "Mark as unread" currently works for me, so I tested this PR and it works fine 👍 


https://github.com/user-attachments/assets/a406223a-247c-4db3-84c6-12f9606fba69